database/models.py

```diff
@@ -97,9 +97,26 @@ class PlanProposition(PlanBase):
     )
 
     plan_regulation_group = relationship(
-        "PlanRegulationGroup", back_populates="plan_regulations"
+        "PlanRegulationGroup", back_populates="plan_propositions"
     )
     text_value: Mapped[language_str]
     ordering: Mapped[autoincrement_int]
     # plan_theme: kaavoitusteema-koodilista
     # ElinkaaritilaX_pvm
+
+
+class SourceData(VersionedBase):
+    """
+    Lähtötietoaineistot
+    """
+
+    __tablename__ = "source_data"
+
+    type_of_source_data_id: Mapped[uuid.UUID] = mapped_column(
+        ForeignKey("codes.type_of_source_data.id", name="type_of_source_data_id_fkey")
+    )
+
+    type_of_source_data = relationship("TypeOfSourceData", back_populates="source_data")
+    name: Mapped[language_str]
+    language: Mapped[language_str]
```

Comment:
Don't know what the language field is for. However, probably it just means the name of the language (e.g. "fin" or "eng" or "swe"). So it should just be a str, not language_str.

Reply:
@msorvoja Okay let's just remove the whole field. Apparently it is a remnant no longer needed in Ryhti.